### ISSUE TITLE
Use mobilenetv2 as backbone train in InterHand2.6M .Why checkpoint so big and Loss is 3.3 not convergence

### ISSUE THREAD

**Issue by NeekHua** [open-mmlab/mmpose#1080]
My train config is following:

_base_ = ['../../../../_base_/datasets/interhand3d.py']
log_level='INFO'
load_from=None
resume_from = None
dist_params = dict(backend='nccl')
workflow = [('train', 1)]
checkpoint_config=dict(interval=5)
evaluation = dict(
    interval=1,
    metric=['MRRPE', 'MPJPE', 'Handedness_acc'],
    save_best='MPJPE_all')
optimizer = dict(
    type='Adam',
    lr=2e-4,
)
optimizer_config = dict(grad_clip=None)
lr_config = dict(policy='step', step=[15, 17])
total_epochs= 20
log_config = dict(
    interval=20,
    hooks=[
        dict(type='TextLoggerHook'),
        # dict(type='TensorboardLoggerHook')
    ])

channel_cfg = dict(
    num_output_channels=42,
    dataset_joints=42,
    dataset_channel=[list(range(42))],
    inference_channel=list(range(42)))

model = dict(
    type='Interhand3D',
    pretrained="mmcls://mobilenet_v2",
    backbone=dict(type='MobileNetV2', widen_factor=1., out_indices=(7, )),
    keypoint_head=dict(
        type='Interhand3DHead',
        keypoint_head_cfg=dict(
            in_channels=1280,
            out_channels=21 * 64,
            depth_size=64,
            num_deconv_layers=3,
            num_deconv_filters=(256, 256, 256),
            num_deconv_kernels=(4, 4, 4),
        ),
        root_head_cfg=dict(
            in_channels=1280,
            heatmap_size=64,
            hidden_dims=(512, ),
        ),
        hand_type_head_cfg=dict(
            in_channels=1280,
            num_labels=2,
            hidden_dims=(512, ),
        ),
        loss_keypoint=dict(type='JointsMSELoss', use_target_weight=True),
        loss_root_depth=dict(type='L1Loss', use_target_weight=True),
        loss_hand_type=dict(type='BCELoss', use_target_weight=True),
    ),
    train_cfg={},
    test_cfg=dict(flip_test=False))

data_cfg = dict(
    image_size=[256, 256],
    heatmap_size=[64, 64, 64],
    heatmap3d_depth_bound=400.0,
    heatmap_size_root=64,
    root_depth_bound=400.0,
    num_output_channels=channel_cfg['num_output_channels'],
    num_joints=channel_cfg['dataset_joints'],
    dataset_channel=channel_cfg['dataset_channel'],
    inference_channel=channel_cfg['inference_channel'])

train_pipeline = [
    dict(type='LoadImageFromFile'),
    dict(type='HandRandomFlip', flip_prob=0.5),
    dict(type='TopDownRandomTranslation', trans_factor=0.15),
    dict(
        type='TopDownGetRandomScaleRotation',
        rot_factor=45,
        scale_factor=0.25,
        rot_prob=0.6),
    dict(type='TopDownAffine'),
    dict(type='ToTensor'),
    dict(
        type='NormalizeTensor',
        mean=[0.485, 0.456, 0.406],
        std=[0.229, 0.224, 0.225]),
    dict(
        type='MultitaskGatherTarget',
        pipeline_list=[
            [dict(
                type='Generate3DHeatmapTarget',
                sigma=2.5,
                max_bound=255,
            )], [dict(type='HandGenerateRelDepthTarget')],
            [
                dict(
                    type='RenameKeys',
                    key_pairs=[('hand_type', 'target'),
                               ('hand_type_valid', 'target_weight')])
            ]
        ],
        pipeline_indices=[0, 1, 2],
    ),
    dict(
        type='Collect',
        keys=['img', 'target', 'target_weight'],
        meta_keys=[
            'image_file', 'joints_3d', 'joints_3d_visible', 'center', 'scale',
            'rotation', 'flip_pairs'
        ]),
]

data_root = '/home/xjh/xujh/data/hand_gesture/InterHand2.6M/InterHand2.6M_5fps_batch1'
data = dict(
    samples_per_gpu=32,
    workers_per_gpu=1,
    val_dataloader=dict(samples_per_gpu=32),
    test_dataloader=dict(samples_per_gpu=32),
    train=dict(
        type='InterHand3DDataset',
        ann_file=f'{data_root}/annotations/human_annot/'
        'InterHand2.6M_train_data.json',
        camera_file=f'{data_root}/annotations/human_annot/'
        'InterHand2.6M_train_camera.json',
        joint_file=f'{data_root}/annotations/human_annot/'
        'InterHand2.6M_train_joint_3d.json',
        img_prefix=f'{data_root}/images/train/',
        data_cfg=data_cfg,
        use_gt_root_depth=True,
        rootnet_result_file=None,
        pipeline=train_pipeline,
        dataset_info={{_base_.dataset_info}}),
)

My train sh is :
launch.py 
--nproc_per_node=3 
--master_port=28901 
tools/train.py 
configs/hand/3d_kpt_sview_rgb_img/internet/interhand3d/mobilenetv2_interhand3d_224x224.py
--launcher pytorch

My train log part of  is:

{"mode": "train", "epoch": 18, "iter": 1000, "lr": 0.0, "memory": 9584, "data_time": 0.14151, "hand_loss": 3.37427, "rel_root_loss": 0.29447, "hand_type_loss": 0.00594, "acc_classification": 0.98977, "loss": 3.67468, "time": 2.96296}
{"mode": "train", "epoch": 18, "iter": 1020, "lr": 0.0, "memory": 9584, "data_time": 0.08183, "hand_loss": 3.36731, "rel_root_loss": 0.24788, "hand_type_loss": 0.00749, "acc_classification": 0.98946, "loss": 3.62268, "time": 2.93801}
{"mode": "train", "epoch": 18, "iter": 1040, "lr": 0.0, "memory": 9584, "data_time": 0.23852, "hand_loss": 3.35966, "rel_root_loss": 0.2912, "hand_type_loss": 0.005, "acc_classification": 0.99004, "loss": 3.65586, "time": 2.91304}
{"mode": "train", "epoch": 18, "iter": 1060, "lr": 0.0, "memory": 9584, "data_time": 0.21366, "hand_loss": 3.32214, "rel_root_loss": 0.26822, "hand_type_loss": 0.00477, "acc_classification": 1.0, "loss": 3.59514, "time": 2.93137}
{"mode": "train", "epoch": 18, "iter": 1080, "lr": 0.0, "memory": 9584, "data_time": 0.14935, "hand_loss": 3.3097, "rel_root_loss": 0.29618, "hand_type_loss": 0.00346, "acc_classification": 0.99706, "loss": 3.60934, "time": 2.99863}
{"mode": "train", "epoch": 18, "iter": 1100, "lr": 0.0, "memory": 9584, "data_time": 0.00063, "hand_loss": 3.30061, "rel_root_loss": 0.29181, "hand_type_loss": 0.00485, "acc_classification": 0.9931, "loss": 3.59727, "time": 3.00785}
{"mode": "train", "epoch": 18, "iter": 1120, "lr": 0.0, "memory": 9584, "data_time": 0.04822, "hand_loss": 3.29228, "rel_root_loss": 0.26536, "hand_type_loss": 0.00456, "acc_classification": 0.99393, "loss": 3.5622, "time": 2.99687}
{"mode": "train", "epoch": 18, "iter": 1140, "lr": 0.0, "memory": 9584, "data_time": 0.25708, "hand_loss": 3.31706, "rel_root_loss": 0.2473, "hand_type_loss": 0.00308, "acc_classification": 1.0, "loss": 3.56744, "time": 2.9175}
{"mode": "train", "epoch": 18, "iter": 1160, "lr": 0.0, "memory": 9584, "data_time": 0.44378, "hand_loss": 3.3656, "rel_root_loss": 0.2865, "hand_type_loss": 0.00566, "acc_classification": 0.9931, "loss": 3.65777, "time": 2.94339}
{"mode": "train", "epoch": 18, "iter": 1180, "lr": 0.0, "memory": 9584, "data_time": 0.79821, "hand_loss": 3.37783, "rel_root_loss": 0.31294, "hand_type_loss": 0.00426, "acc_classification": 1.0, "loss": 3.69503, "time": 2.99389}
{"mode": "train", "epoch": 18, "iter": 1200, "lr": 0.0, "memory": 9584, "data_time": 0.60982, "hand_loss": 3.51285, "rel_root_loss": 0.30585, "hand_type_loss": 0.00561, "acc_classification": 0.99667, "loss": 3.82431, "time": 3.0674}
{"mode": "train", "epoch": 18, "iter": 1220, "lr": 0.0, "memory": 9584, "data_time": 0.44942, "hand_loss": 3.46613, "rel_root_loss": 0.34197, "hand_type_loss": 0.00237, "acc_classification": 0.9975, "loss": 3.81046, "time": 2.99734}
{"mode": "train", "epoch": 18, "iter": 1240, "lr": 0.0, "memory": 9584, "data_time": 0.4275, "hand_loss": 3.34551, "rel_root_loss": 0.28015, "hand_type_loss": 0.00586, "acc_classification": 0.99373, "loss": 3.63151, "time": 2.92361}
{"mode": "train", "epoch": 18, "iter": 1260, "lr": 0.0, "memory": 9584, "data_time": 0.44012, "hand_loss": 3.54345, "rel_root_loss": 0.33964, "hand_type_loss": 0.00525, "acc_classification": 0.99456, "loss": 3.88834, "time": 3.01803}
{"mode": "train", "epoch": 18, "iter": 1280, "lr": 0.0, "memory": 9584, "data_time": 0.32151, "hand_loss": 3.46035, "rel_root_loss": 0.3304, "hand_type_loss": 0.00235, "acc_classification": 0.99722, "loss": 3.7931, "time": 3.03429}
{"mode": "train", "epoch": 18, "iter": 1300, "lr": 0.0, "memory": 9584, "data_time": 0.16789, "hand_loss": 3.32896, "rel_root_loss": 0.29423, "hand_type_loss": 0.00567, "acc_classification": 0.99286, "loss": 3.62885, "time": 3.03294}
{"mode": "train", "epoch": 18, "iter": 1320, "lr": 0.0, "memory": 9584, "data_time": 0.12259, "hand_loss": 3.36471, "rel_root_loss": 0.32608, "hand_type_loss": 0.00378, "acc_classification": 0.99583, "loss": 3.69457, "time": 2.98504}

My train model so big :have 200MB
![image](https://user-images.githubusercontent.com/32665609/146109941-cd510553-6daf-4913-af5c-9e2842406a21.png)

What should I do ?Any suggestions would be greatly appreciated

**Comment by NeekHua** [comment]
I got it
